backend/repository/config.go

```diff
@@ -72,7 +72,7 @@ type EndpointConfig struct {
 	ClientID         string                                  `json:"clientId"`
 	ClientMethod     string                                  `json:"clientMethod"`
 	TestFixtures     map[string]*codegen.EndpointTestFixture `json:"testFixtures"`
-	Middlewares      []*EndpointMiddlewareConfig             `json:"middlewares"`
+	Middlewares      []EndpointMiddlewareConfig              `json:"middlewares"`
```

Comment:
middlewares can be nil and crashes [here](https://github.com/uber/zanzibar/blob/master/backend/repository/endpoint_update.go#L201)